py/desisim/scripts/quickbrick.py

```diff
@@ -120,9 +120,18 @@ def main(args):
     log.debug('Using OBJTYPE {}'.format(objtype))
     if objtype == 'BGS' or objtype == 'MWS' or objtype == 'BRIGHT_MIX':
         qsim.observation.exposure_time = desiparams['exptime_bright'] * u.s
-        qsim.atmosphere.moon.moon_zenith = args.moon_zenith * u.deg
-        qsim.atmosphere.moon.separation_angle = args.moon_angle * u.deg
-        qsim.atmosphere.moon.moon_phase = args.moon_phase
+        if args.moon-phase is None:
+            qsim.atmosphere.moon.moon_phase = 0.7
+        else:
+            qsim.atmosphere.moon.moon_phase = args.moon-phase
```

Comment:
A quickbrick test with flavor=BGS would catch the typos of `args.moon-phase` instead of `args.moon_phase`.  That is more than just a naming convention: `args.moon-phase` would be trying to subtract two non-existence variables `args.moon` and `phase` and would have crashed.  You have fixed that now; good.

I see that we do have quickbrick tests for flavor='BGS' but Travis was skipping them because memory issues with specsim.  Those have also now been fixed with specsim master, so I'll go ahead and merge this now and we can separately enable the quickbrick and quickgen tests for Travis.